includes/jobs/GlobalNewFilesDeleteJob.php

```diff
@@ -0,0 +1,36 @@
+<?php
```

Comment:
Returns `Serialization of 'Closure' is not allowed` and I'm not sure why.

Reply:
Hmm, you may have to remove `$file` as a property of the class. Instead of accessing `$this->file`, access `$this->params['file']`. I don't really understand the problem either though, so I'm not 100% certain what the solution here is.

Reply:
made the local change on test3, that did not fix it. Same error.

Reply:
This job (not yet GlobalNewFilesInsertJob) is now fixed.